plugins/ignoreMessagesBeforeStart.js

```diff
@@ -1,17 +1,26 @@
-var startDate;
+const startDate = {};
 
 module.exports = {
 
     id: 'ignoreMessagesBeforeStart',
 
-    plugin(bot) {
+    defaultConfig: {
+        start: true
+    },
 
-        bot.on(['start', '/start'], () => {
-            startDate = Date.now();
+    plugin(bot, config) {
+
+        bot.on([config.start ? 'start', '/start' : 'start'], (msg) => {
```

Comment:
Error on this line, please fix.

Reply:
Whats the error?

Reply:
/home/travis/build/mullwar/telebot/plugins/ignoreMessagesBeforeStart.js
  13:39  error  Parsing error: Unexpected token ,